### PR TITLE
Exam

### PR DESCRIPTION
**Pull request by AlinaDubchak**
Aded and optimized tasks (inc.js and ip.js)

### CODE REVIEW

File: Tasks/inc.js [hunk 1]
```diff
@@ -1,14 +1,15 @@
-// Increment all numbers in dictionary
+'use strict'
```
Comment (tshemsedinov) [inline Tasks/inc.js:1]: - Use eslint and prettier

File: Tasks/inc.js [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
 
-let inc_numbers=(
```
Comment (tshemsedinov) [inline Tasks/inc.js:3]: - Prefer const
- Use round brackets even for single argument lambda functions

File: Tasks/inc.js [hunk 3]
```diff
@@ -1,14 +1,15 @@
...
-  (format_complete,...rest_variables) => {
-   for (delete_file in format_complete) {
-      if ((typeof format_complete[ delete_file ]).charAt(0).toUpperCase() ==='N'){
-      format_complete[ delete_file ] = format_complete[delete_file] + 1}
+const inc = elements => {
```
Comment (tshemsedinov) [inline Tasks/inc.js:3]: Use round brackets even for single argument lambda functions

File: Tasks/inc.js [hunk 4]
```diff
@@ -1,14 +1,15 @@
...
+     for ( let key in elements){
+      if ((typeof elements[key]) == "number")
```
Comment (tshemsedinov) [inline Tasks/inc.js:5]: Never use `==`

File: Tasks/inc.js [hunk 5]
```diff
@@ -1,14 +1,15 @@
-// Increment all numbers in dictionary
+'use strict'
 
-let inc_numbers=(
-  (format_complete,...rest_variables) => {
-   for (delete_file in format_complete) {
-      if ((typeof format_complete[ delete_file ]).charAt(0).toUpperCase() ==='N'){
-      format_complete[ delete_file ] = format_complete[delete_file] + 1}
+const inc = elements => {
+     for ( let key in elements){
```
Comment (tshemsedinov) [inline Tasks/inc.js:4]: Prefer for..of

File: Tasks/inc.js [hunk 6]
```diff
@@ -1,14 +1,15 @@
...
+      if ((typeof elements[key]) == "number")
+      {
+        elements[key] = elements[key] + 1;
```
Comment (tshemsedinov) [inline Tasks/inc.js:7]: ++

File: Tasks/inc.js [hunk 7]
```diff
@@ -1,14 +1,15 @@
...
+      }
    }
-  return format_complete;
-  }
-)
+  return elements;
```
Comment (tshemsedinov) [inline Tasks/inc.js:10]: Don't change incoming parameters

**Review by tshemsedinov — COMMENTED** [review]
Terrible formatting, plz fix all linter warnings

**Comment by AlinaDubchak** [comment]
// another implementation using forEach.
'use strict';

const inc = format => {
  format.forEach((element) => {
  if(typeof element === 'number') element ++
  console.log(element);
  })
};

const obj = [1, 2, 'hello',false];

inc(obj);

**Comment by AlinaDubchak** [comment]
// another implementation using map.
'use strict';

const inc = (format) =>{
    let res = format.map(function(element){
    if (typeof element === 'number') element++ 
    return element; 
    })
    return res;
}

const obj = [1, 2, 'hello', false, 4, 'mouse'];

console.log(inc(obj));

**Pull request closed without merging**